### ISSUE TITLE
[feature request] Custom TOTP Seed/Settings Field Name

### ISSUE THREAD

**Issue by tty7tyil** [Kunzisoft/KeePassDX#886]
**Is your feature request related to a problem? Please describe.**
In KeeTrayTOTP on Windows we can set the field name for TOTP seed and settings, other client won’t recognize these fields with custom names.  However, Keepass2Android does provide the same settings to set custom field names as to KeeTrayTOTP.

**Describe the solution you'd like**
Add two app setting entries for custom TOTP field names, just like KeeTrayTOTP/Keepass2Android.

**Comment by droidmonkey** [comment]
KeePassXC is not going to be implementing this.

**Comment by J-Jamet** [comment]
This format is already implemented. A bug was fixed in development because the settings were not correctly formatted (will be release in [2.9.13](https://github.com/Kunzisoft/KeePassDX/issues/878)).

I think you are confused because the TOTP form of KeePassDX generates an `otp` field identical to the one of KeePassXC (otpauth link).

I'm not going to generate a new format automatically by this form (at least not a plugin format), if you want to modify the parameters related to this special format, do it manually.

Especially since now there is another native format in KeePass2 since 2.47, which makes this plugin obsolete.

**Comment by J-Jamet** [comment]
I will reconsider the question of choosing the form generation format when I have time. But I have other more urgent matters to deal with at the moment.

**Comment by tty7tyil** [comment]
Thanks for the information about KP native format.

KeePassXC seems to use the Key-Uri format, aren’t KeePassDX using this format too?  (with the same field name `otp`)

**Comment by J-Jamet** [comment]
Yes, the otp by URI (otpauth) format is recognized and the one generated by the form in KeePassDX.

**Comment by tty7tyil** [comment]
I understand that although DX use key-uri format by default, it does support the two field format used by Tray TOTP.  But the original problem I tried to say is that, those two field names can be customized; and once customized it won’t be recognized by other clients unless they provide the ability to let the user specify what field names to look for.

Anyway, this use case of customized field name is probably rare, I have already taken some time to migrate my DB to a more standard format 😂

**Comment by J-Jamet** [comment]
I understand the problem very well, that's why I recommend that you create custom fields manually if you want another format instead of using KeePassDX's OTP generator form.

I can improve the concept by providing the same output format as the input if using the form.
It can be done for one entry but then we have to generalize it to the whole database to make it consistent.

Like I said, I don't have time to do this functionality right now.
